wip

hexfusion opened the pull request:
Signed-off-by: Sam Batschelet <sbatsche@redhat.com>

Commit:
bootkube.sh: wait for cluster-etcd-operator to finish before declaring success

alaypatel07 commented on cmd/openshift-install/create.go:
```diff
@@ -105,19 +105,19 @@ var (
 					logrus.Fatal(err)
 				}
+
+				logrus.Info("Destroying the bootstrap resources...")
```
With the subcommand, we can take this to line 108 where it originally was.

hexfusion committed:
bootkube.sh: add cluster-etcd-operator support

Signed-off-by: Sam Batschelet <sbatsche@redhat.com>

hexfusion committed:
bootkube.sh: add conditional logic to allow disabling cluster-etcd-operator

This PR is a temporary solution that allows CEO to be deployed in an Unmanaged state. This allows us to test
the bootstrap process and rolling out of the operator with the controllers performing no-op. This PR also covers
the case where CEO needs to be removed from the CVO operator payload completely. In which case we will fall back
to the existing SRV bootstrap for etcd.

Signed-off-by: Sam Batschelet <sbatsche@redhat.com>
